ehsm_kms_service/kubernetes/ehsm-db.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: couchdb
+  namespace: intel-ehsm

```

Comment:
can we change the name to "ehsm-kms", don't want to use the intel-xxx

Reply:
It will be change